### DISCUSSION TITLE
Is associated_collection_scope config different for version 3.0.0?

### DISCUSSION THREAD

**Discussion by darrenterhune** [railsadminteam/rails_admin#3525] [Q&A]
I originally would have every model config in the initializer but due to some deployment issues (specifically assets:precompile, which happens in a sandbox instance) we can not longer use the initializer to config. So we moved it all to concerns for each model. However upon doing so we can not longer use `associated_collection_scope` like:

```ruby
class User < ApplicationRecord
  has_many :funds_users, dependent: :destroy
  has_many :funds, through: :funds_users
  
  rails_admin do
    edit do
      field :funds do
        associated_collection_scope {}
      end
    end
  end
end
```

It raises an error

```
undefined method `associated_collection_scope' for #<RailsAdmin::Config::Fields::Types::String[funds] @parent=RailsAdmin::Config::Sections::Edit, @root=RailsAdmin::Config::Model, @abstract_model=#<RailsAdmin::AbstractModel:0x00007ff489a275d0 @model_name="User", @adapter=:active_record>, @defined=true, @name=:funds, @order=22, @properties=nil, @section=RailsAdmin::Config::Sections::Edit, @eager_load_registered=true>
```

All the documentation seems to claim that this should work 🤷‍♂️ 

It appears that maybe per model config has loading issues as described here https://github.com/railsadminteam/rails_admin/pull/3492

**Comment by mshibuya** [comment]
It's detected as a String field, not an association. How is `funds` configured in the model?

**Reply by darrenterhune** [reply]
Updated the post. After more debugging I think this is a loading issue as described in #3492 some requests work, some don't. Initial page load breaks subsequent refreshes load and some break. I had originally thought maybe the docs were out of date and that something changed with how to use `associated_collection_scope`.
